utest/run.py

```diff
@@ -21,12 +28,12 @@
 
 
 if not sys.warnoptions:
-    warnings.simplefilter('always')
+    warnings.simplefilter("always")
 
 
 base = os.path.abspath(os.path.normpath(os.path.split(sys.argv[0])[0]))
-for path in ['../src', '../atest/testresources/testlibs']:
-    path = os.path.join(base, path.replace('/', os.sep))
+for path in ["../src", "../atest/testresources/testlibs"]:
+    path = os.path.join(base, path.replace("/", os.sep))
```

Comment:
Changing quotes is a bit problematic for these reasons:
- We prefer `'` over `"` in general.
- This change has nothing to do with the actual fix or even with `utest/run.py` enhancements.

Not a huge problem and assuming there are no other things to change in this PR we can leave these.

Reply:
Annoyingly the latest version of black is responsible for these changes

Reply:
Yeah, Black prefers `"` instead of `'` but that can be configured. I wouldn't in general use code formatters when creating a PR but you can also select what changes you commit. Anyway, not a big issue this. Although we generally use `'`, there's code using `"` as well.

Reply:
Would it be worth adding a pyproject.toml file with this setting to the project so that the issue isnt hit going forward?

Reply:
I've reverted the change but haven't included a pyproject file in this pr